tracing/opentelemetry/src/main/java/io/helidon/tracing/opentelemetry/OpenTelemetrySpan.java

```diff
@@ -104,7 +104,7 @@ public Span baggage(String key, String value) {
         Baggage.builder()
                 .put(key, value)
                 .build()
-                .storeInContext(getContext())
+                .storeInContext(getContext().with(delegate))
```

Comment:
wrong formatting, please keep same code style (`.with(delegate)` should be on a new line)

Reply:
done